demo/test.sh

```diff
@@ -3,6 +3,8 @@
 set -eux
 
 ./dev.sh start
+
+REDIRECT_URI=http://test.sso.example.com:8080/login-callback.html
```

Comment:
why did the redirect_uri changed from http://test.sso.example.com to  http://test.sso.example.com:8080/login-callback.html?

Reply:
- The current redirect_uri is not work in tesh.sh
- Redirect_uri in lemonLdap is http://test.sso.example.com:8080/login-callback.html (https://github.com/linagora/tmail-backend/blob/9367e78404f5ae82750cd3ca64866012ed2ae8e1/demo/lemonldap/lmConf-1.json#L222)